### ISSUE TITLE
gource gives error "Access is denied\n gource: failed to generate log file" after Win 10 update to 1809 (using Mercurial)

### ISSUE THREAD

**Issue by SudoCerb** [acaudwell/Gource#199]
Hi,

I've seen a couple of issues similar to this, but I successfully ran gource this very morning prior to updating to the latest Windows 10 Feature Update :-/

I have not knowingly made any other changes on my system.

I can execute `hg log` just fine. Any suggestions?

**Comment by SudoCerb** [comment]
I tried cloning my repo into another directory too, to see if that would work, but to no avail.

**Comment by SudoCerb** [comment]
Of course, running it on my Ubuntu 19.04 machine works just fine :nerd_face: compiled from source and watched the magic.

Looking at the Mercurial log file between the two, it looks like part of my name got corrupted on the Windows machine.

Still trying to figure it out.

**Comment by acaudwell** [comment]
How if your name corrupted? Maybe its a character encoding issue.

You could manually generate the log using the command output by `gource --hg-log-command` edit the name and see if that's the problem.
